modules/cron/main.tf

```diff
@@ -46,6 +46,16 @@ resource "ko_build" "image" {
   repo        = local.repo
 }
 
+resource "google_cloud_run_v2_job_iam_member" "invoker" {
```

Comment:
Should this also exempt the invoker from the [laser grid](https://github.com/chainguard-dev/terraform-infra-common/blob/main/modules/cron/main.tf#L192)?

Reply:
good point, added exclusion